bin/generate-tests.php

```diff
@@ -91,7 +91,7 @@
 
 add_test(['parallel:sleep', '--sleep5', '0', '--sleep7', '0', '--sleep10', '0'], 'ParallelSleepTest');
 add_test(['context:context', '--context', 'run'], 'ContextContextRunTest');
-add_test(['context:context', '--context', 'my_default', '-vvv'], 'ContextContextMyDefaultTest');
+add_test(['context:context', '--context', 'my_default', '-vv'], 'ContextContextMyDefaultTest');
```

Comment:
Adding logs to the EventManager class adds logs to the output when it's at maximum verbosity and this records the time and breaks the tests, is there a solution for this?

Reply:
For testing log (should we really test that? I don't thinks so), I don't use the regular toolchain. See https://github.com/jolicode/castor/blob/main/tests/Examples/LogInfoTest.php for example.

Reply:
For testing log (should we really test that? I don't thinks so), I don't use the regular toolchain. See https://github.com/jolicode/castor/blob/main/tests/Examples/LogInfoTest.php for example.

Reply:
I'll take care of it. Ping me when the PR is ready, I'll finish it